manim/mobject/mobject.py

```diff
@@ -1011,9 +1011,11 @@ def scale(self, scale_factor: float, **kwargs) -> "Mobject":
             return self
 
     def rotate_about_origin(self, angle, axis=OUT, axes=[]):
+        """Rotates an mobject about the ORIGIN, which is at [0,0,0]"""
         return self.rotate(angle, axis, about_point=ORIGIN)
 
     def rotate(self, angle, axis=OUT, **kwargs):
+        """Rotates an mobject about a certain point e.g. ``m.rotate(PI, about_point = 2*LEFT)``"""
```

Comment:
This is a bit inconvenient:
![image](https://user-images.githubusercontent.com/44469195/113157649-6038e800-923b-11eb-9450-a30f1fdec40d.png)
but later the code is rendered:
![image](https://user-images.githubusercontent.com/44469195/113157682-69c25000-923b-11eb-95f8-fcdf15acb2a8.png)
So should I remove the code ``m.rotate(PI, about_point = 2*LEFT)`` completely or make a new line?

Reply:
The oneline summary is not intended for examples. If you want to add an example, do so in a new line (preferably as a doctest then).